archive/s/swift/reverse-string.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+guard CommandLine.arguments.count > 0 else {
+    exit(0)
+}
+
+let usersString = CommandLine.arguments[0]
+let reversedCollection = usersString.reversed()
```

Comment:
In other [online editors](https://www.jdoodle.com/execute-swift-online), I'm finding that they can't find `reversed()`.

```
jdoodle.swift:8:26: error: value of type 'String' has no member 'reversed'
let reversedCollection = usersString.reversed()
```

Reply:
I think that's due to them running an earlier version of Swift. Swift4 changed the name of the method from reverse to reversed. Similarly, earlier versions did not have access to a count on String -- you have to call count on the string's characters property

Reply:
Good to know! I'll make a note of that in the article.